api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -465,6 +465,158 @@ function rewardReducer(network, validators, reward, reducers) {
   return parsedRewards
 }
 
+function democracyProposalReducer(
+  network,
+  proposal,
+  totalIssuance,
+  blockHeight
+) {
+  return {
```

Comment:
ok for now, but we want to have a different structure in the future: https://github.com/luniehq/lunie/issues/4445 like `proposalsV2`

Reply:
we will do this in the old format and upgrade in a later PR